vpr/src/util/vpr_utils.cpp

```diff
@@ -905,6 +905,91 @@ bool primitive_type_feasible(const AtomBlockId blk_id, const t_pb_type* cur_pb_t
     //Feasible
     return true;
 }
+bool primitive_type_feasible(const AtomBlockId blk_id, t_pb_graph_node* curr_pb_graph_node) {
```

Comment:
Need a blank line above this (but maybe make format will put it in).

Reply:
Added blank line